app/controllers/admin/organizations_controller.rb

```diff
@@ -0,0 +1,103 @@
+class Admin::OrganizationsController < Admin::BaseController
+  before_action :authenticate_user!, except: [:index, :show, :filter]
+  before_action :find_organization, only: [:show, :edit, :update, :destroy]
+
+  ORGANIZATIONS_PER_PAGE = 18
+
+  def new
+    if current_user.organization.present?
+      redirect_to organization_path(current_user.organization_id), alert: "You can only have one organization."
+    end
+    @organization = Organization.new
+  end
+
+  def show
+    @claimed = @organization.claim_requests.find_by_status(true)
+    @news_articles = @organization.news_articles
+    hosting_event
+
+    respond_to do |format|
+      format.html { render }
+      format.json { render json: @organization.to_json }
+    end
+  end
+
+  def index
+    @technologies = Technology.all
+    if current_user && user_is_admin?
```

Comment:
I noted this on another PR. Why are we checking that the user is admin here? It should be the case for all admin users. Can you please check that this controller isn't used elsewhere?